AndroidSDKCore/src/main/java/com/leanplum/monitoring/LeanplumCrashHandler.java

```diff
@@ -0,0 +1,41 @@
+package com.leanplum.monitoring;
+
+import android.content.Context;
+
+import com.leanplum.internal.Log;
+
+import java.lang.reflect.Constructor;
+
+public class LeanplumCrashHandler {
```

Comment:
the class name can be just `CrashHandler`, it already lives in package `com.leanplum.monitoring`